pkg/daemon/ceph/config/info.go

```diff
@@ -63,7 +64,9 @@ func NewMonInfo(name, ip string, port int32) *MonInfo {
 func (c *ClusterInfo) Log(logger *capnslog.PackageLogger) {
 	mons := []string{}
 	for _, m := range c.Monitors {
-		mons = append(mons, fmt.Sprintf("{Name: %s, Endpoint: %s}", m.Name, m.Endpoint))
+		input := ("{Name: " + m.Name + ", Endpoint: " + m.Endpoint + "}")
+		encoded := base64.StdEncoding.EncodeToString([]byte(input))
```

Comment:
If we did need to ensure safe encoding, consider:
- We wouldn't want base64. We would just want to escape the invalid characters.
- Only input variables would potentially need to be escaped, such as `m.Name` or `m.Endpoint`.